templates/dhcpd.pool.erb

```diff
@@ -16,6 +16,12 @@ subnet <%= @network %> netmask <%= @mask %> {
 <% if @gateway && !@gateway.strip.empty? -%>
   option routers <%= @gateway %>;
 <% end -%>
+<% if @static_routes
+  @static_routes.each do |static_route| -%>
```

Comment:
could you please write
```erb
<% if @static_routes -%>
<%   @static_routes.each do |static_route| -%>
...
<%   end -%>
<% end -%>
```
or similar here, I was just confused by seeing to ```<% end -%>```s and thought of a duplicate line.